02-peer-protocol.md

```diff
@@ -3296,12 +3296,17 @@ messages are), they are independent of requirements here.
     1. type: 0 (`next_funding`)
     2. data:
         * [`sha256`:`next_funding_txid`]
-    1. type: 1 (`your_last_funding_locked`)
-    2. data:
-        * [`sha256`:`your_last_funding_locked_txid`]
-    1. type: 3 (`my_current_funding_locked`)
+    1. type: 5 (`my_current_funding_locked`)
     2. data:
         * [`sha256`:`my_current_funding_locked_txid`]
+        * [`byte`:`retransmit_flags`]
```

Comment:
Do we want this always tied to `my_current_funding_locked_txid`? In other words, is there any future use where a flag is independent of that?